app/lib/common/widgets/html_editor/html_editor.dart

```diff
@@ -404,11 +446,28 @@ class HtmlEditorState extends State<HtmlEditor> {
     if (attributes == null) {
       return before;
     }
+
     final roomId = widget.roomId;
     // Inline Mentions
-    final mention = attributes.entries
-        .firstWhere((e) => e.value is MentionAttributes)
-        .value as MentionAttributes?;
+    MentionAttributes? mention;
```

Comment:
This fix solves the gray area / bad state: exception problem because of nullable object here and now we have wrapped with proper error handling and return fallback in case, it fails.